Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -1747,6 +1760,7 @@ localizedStrings["Update Local Override"] = "Update Local Override";
 localizedStrings["Usage: %s"] = "Usage: %s";
 localizedStrings["Use Mock Capture Devices"] = "Use Mock Capture Devices";
 localizedStrings["Use default media styles"] = "Use default media styles";
+localizedStrings["Use mock capture devices"] = "Use mock capture devices";
```

Comment:
oops?

Reply:
These three strings match the intended capitalization, however it is strange that the previous capitalizations continue to exist.

Reply:
I had used the script to automatically extract strings. Didn't occur to me it wouldn't also clean-up unused strings. 
Cleaned up manually in the latest patch.